src/components/EmailForm/index.tsx

```diff
@@ -37,79 +36,56 @@ export const EmailForm: React.FC<EmailFormProps> = ({ eventName, mail }) => {
     formState: { errors },
   } = formProps;
 
-  const onSubmit = async (body: EmailBody) => {
-    const subject = `Registrace na sportovní akci ${eventName}`;
+  const onSubmit = async (text: Text) => {
```

Comment:
I would call it emailValues or emailData. The text is a bit misleading, sounds like some primitive/string

Reply:
Done.